config/packages/security.yaml

```diff
@@ -2,31 +2,20 @@ security:
     access_decision_manager:
         strategy: unanimous
         allow_if_all_abstain: true
+
     # https://symfony.com/doc/current/security.html#registering-the-user-hashing-passwords
     password_hashers:
         Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
+
     # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
     providers:
         sulu:
             id: sulu_security.user_provider
-    # Easy way to control access for large sections of your site
-    # Note: Only the *first* access control that matches will be used
-    access_control:
-        - { path: ^/admin/reset, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/security/reset, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/login$, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/2fa, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/_wdt, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/_profiler, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/translations, roles: PUBLIC_ACCESS }
-        - { path: ^/admin$, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/$, roles: PUBLIC_ACCESS }
-        - { path: ^/admin/p/, roles: PUBLIC_ACCESS }
-        - { path: ^/admin, roles: ROLE_USER }
```

Comment:
moved this below to match Symfony Security yaml and make it easier for flex in future update create the `diff`